src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -139,7 +141,11 @@ public function __construct(
             we need the base dir here */
         $dataDirectory = dirname($this->dataDirectory);
 
-        $workspaceProviderFactoryFactory = new WorkspaceProviderFactoryFactory($this->logger, $this->clientWrapper);
+        $workspaceProviderFactoryFactory = new WorkspaceProviderFactoryFactory(
+            new Components($this->clientWrapper->getBranchClient()),
```

Comment:
Tady me trochu zarazilo, ze jsme se bavili porad o Workspaces API, ale musle jsme to v tohle pripade dat i ke Components, jinak mi testy neprosly.